components/collapsible-panel/README.md

```diff
@@ -389,11 +387,14 @@ Use the `d2l-collapsible-panel-group` component to automatically handle spacing
 
 ## Accessibility
 
+The `d2l-collapsible-panel` component adheres to [W3C's Disclosure Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/) and any components added to any of the slots should also adhere to its respective pattern.
+
 ### Panel label
+
 By default, the panel is described by screen readers with the `panel-title` attribute. There may be situations where the screen reader should read additional information. In this case, a special label can be specified using the `expand-collapse-label` property.
 
 ### Keyboard behaviour
 
-On focus, a focus ring (blue border) goes around the clickable area of the component. When collapsed, this area is the entire panel; when expanded, it's only the header.
+On focus, a focus ring (blue border) goes around the clickable area of the component. When collapsed, this area is the entire panel; when expanded, it's only the header. Once focused, using the `Enter` and `Space` keys, allows the user to toggle between the show and hide states.
```

Comment:
Finding that last sentence a bit awkward with the commas. Here's an alternative:

When focused, users can expand or collapse the panel using the `Enter` or `Space` key.